src/server/game/Server/Packets/QuestPackets.h

```diff
@@ -108,6 +110,62 @@ namespace WorldPackets
 
                 QuestInfo Info;
         };
+
+        struct QuestChoiceItem
+        {
+            QuestChoiceItem(uint32 itemID, uint32 quantity, uint32 displayID) : ItemID(itemID), Quantity(quantity),
+                                                                                DisplayID(displayID) { }
+            uint32 ItemID = 0;
+            uint32 Quantity = 0;
+            uint32 DisplayID = 0;
+        };
+
+        struct QuestRewards
+        {
+            std::vector<QuestChoiceItem> UnfilteredChoiceItems;
+            std::vector<QuestChoiceItem> RewardItems;
+            uint32 RewardMoney = 0;
+            uint32 RewardXPDifficulty = 0;
+            uint32 RewardHonor = 0;
+            float RewardKillHonor = 0.f;
+            uint32 RewardDisplaySpell = 0;
+            int32 RewardSpell = 0;
+            uint32 RewardTitleId = 0;
+            uint32 RewardTalents = 0;
+            uint32 RewardArenaPoints = 0;
+            uint32 RewardFactionFlags = 0;
+            std::array<uint32, QUEST_REPUTATIONS_COUNT> RewardFactionID = { };
+            std::array<int32, QUEST_REPUTATIONS_COUNT> RewardFactionValue = { };
+            std::array<int32, QUEST_REPUTATIONS_COUNT> RewardFactionValueOverride = { };
+        };
+
+        struct QuestDescEmote
+        {
+            QuestDescEmote(int32 type, uint32 delay) : Type(type), Delay(delay) { }
+            uint32 Type;
+            uint32 Delay;
+        };
+
+        class QuestGiverQuestDetails final : public ServerPacket
+        {
+        public:
+            QuestGiverQuestDetails() : ServerPacket(SMSG_QUESTGIVER_QUEST_DETAILS) { }
```

Comment:
```suggestion
            QuestGiverQuestDetails() : ServerPacket(SMSG_QUEST_GIVER_QUEST_DETAILS) { }
```

Reply:
and in another places so it is like in other branches https://github.com/TrinityCore/WowPacketParser/blob/6fc34f1deb4d92b385886495ca98c21cc772e71c/WowPacketParser/Parsing/Parsers/QuestHandler.cs#L808